Templates/DefaultComponent/Template/Include/${GemName}/${Name}Interface.h

```diff
@@ -0,0 +1,32 @@
+// {BEGIN_LICENSE}
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+// {END_LICENSE}
+
+#pragma once
+
+#include <AzCore/Component/ComponentBus.h>
+
+namespace ${GemName}
+{
+    class ${SanitizedCppName}Requests
+        : public AZ::ComponentBus
+    {
+    public:
+        AZ_RTTI(${GemName}::${SanitizedCppName}Requests, "{${Random_Uuid}}");
+
+        // Put your public request methods here
+    };
+    using ${SanitizedCppName}RequestBus = AZ::EBus<${SanitizedCppName}Requests>;
+
+    inline namespace ${SanitizedCppName}Interface
+    {
+        inline constexpr auto Get = [](AZ::ComponentBus::BusIdType busId) {return ${SanitizedCppName}RequestBus::FindFirstHandler(busId); };
```

Comment:
```suggestion
        inline constexpr auto Get = [](AZ::ComponentBus::BusIdType busId) { return ${SanitizedCppName}RequestBus::FindFirstHandler(busId); };
```

Reply:
I would still recommend deleting the interface. I don't understand the benefit and it bypasses some of the assurances you get from the bus. Otherwise, looks good and the other things can be fleshed out in future iterations.

Reply:
@gadams3 I'll continue working with @lumberyard-employee-dm on this and we'll either update and clarify the interface or remove it. Also, thanks for the component configuration example! I'll work on adding this in the next iteration of this template.

Reply:
FYI, I've removed the bus interface for now.